core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -545,4 +552,17 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
       this.changeDetectorRef.detectChanges();
     }
   }
+
+  getImageInfoForSuggestion(content: string | string[]): string {
+    let htmlString = '';
+
+    if (!Array.isArray(content)) {
```

Comment:
Why do we check for this? Could you please explain it before the condition by a comment?

Reply:
I have now added a comment to explain what the function does.